inventory/dynamic/gcp/hosts.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
```

Comment:
Consider updating the README to include information about the environmental vars
 ==== REQUIRED ====
GCE_EMAIL
GCE_PEM_FILE_PATH

===== OPTIONAL ==== 
GCE_PROJECT
GCE_ZONE
GCE_INI_PATH
INVENTORY_IP_TYPE
GCE_TAGGED_INSTANCES